source/_integrations/media_player.markdown

```diff
@@ -48,6 +48,79 @@ Available services: `turn_on`, `turn_off`, `toggle`, `volume_up`, `volume_down`,
 | `entity_id`            |      yes | Target a specific media player. To target all media players, use `all`.                                                                                                                       |
 | `media_content_id`     |       no | A media identifier. The format of this is integration dependent. For example, you can provide URLs to Sonos and Cast but only a playlist ID to iTunes.                   |
 | `media_content_type`   |       no | A media type. Must be one of `music`, `tvshow`, `video`, `episode`, `channel` or `playlist`. For example, to play music you would set `media_content_type` to `music`. |
+| `extra`                |      yes | Extra dictionary data to send, e.g., title, thumbnail. Possible values can be found below.
+
+##### Extra dictionary data
+
+{% configuration %}
+title:
+  type: string
+  description: Title of the media.
+  required: false
+thumb:
+  type: string
+  description: Thumbnail image URL.
+  required: false
+current_time:
+  type: float
+  description: Seconds since beginning of content. If the content is live content, and position is not specifed, the stream will start at the live position
+  required: false
+autoplay:
+  type: bool
+  description: Whether the media will automatically play.
+  required: false
+stream_type:
+  type: string
+  description: Describes the type of media artifact as one of the following: "NONE", "BUFFERED", "LIVE".
+  required: false
+subtitles:
+  type: string
+  description: URL of subtitle file to be shown on chromecast.
+  required: false
+subtitles_lang:
+  type: string
+  description: Language for subtitles.
+  required: false
+subtitles_mime:
+  type :string
+  description: Mimetype of subtitles.
+  required: false
+subtitle_id:
+  type: int
+  description: Id of subtitle to be loaded.
+  required: false
+enqueue:
+  type: bool
+  description: If True, enqueue the media instead of play it.
+  required: false
+media_info:
+  type: map
+  description: Additional MediaInformation attributes not explicitly listed.
+  required: false
+metadata:
+  type: map
+  description: Media metadata object, one of the following: GenericMediaMetadata, MovieMediaMetadata, TvShowMediaMetadata, MusicTrackMediaMetadata, PhotoMediaMetadata.
```

Comment:
```suggestion
  description: "Media metadata object, one of the following: `GenericMediaMetadata`, `MovieMediaMetadata`, `TvShowMediaMetadata`, `MusicTrackMediaMetadata`, `PhotoMediaMetadata`."
```

However, since this is a map, it should be listed as key instead.